### ISSUE TITLE
How old are you?

### ISSUE THREAD

**Issue by sarbbottam** [sindresorhus/ama#43]
And how long have you been coding?

**Comment by sindresorhus** [comment]
I'm 26 (89' model) and I've coding since I was 20. Though I've been into computers and doing websites since I was 8.

**Comment by robbbz** [comment]
> Though I've been into computers and doing websites since I was 8.

That's super young dude!

**Comment by sarbbottam** [comment]
Thank you.

**Comment by sindresorhus** [comment]
> That's super young dude!

Website might be an abuse of the word. It was a hot curry of animated 3D gifs, background MIDI music, and trailing cursor star animation, cooked together in Notepad.exe. Good times.

**Comment by sarbbottam** [comment]
>> Website might be an abuse of the word. It was a hot curry of animated 3D gifs, background MIDI music, and trailing cursor star animation

You mean something like http://www.govindtiwari.blogspot.com/? :wink:

**Comment by robbbz** [comment]
> cooked together in Notepad.exe. Good times.

Alright but at 8? The youngest kids I see coding are like 16-17 and they're just toying with processing instead of doing useful stuff. Respect bro. You should be really talented.

**Comment by sindresorhus** [comment]
> You mean something like http://www.govindtiwari.blogspot.com/? :wink:

It was even worse.

> Alright but at 8? The youngest kids I see coding are like 16-17 and they're just toying with processing instead of doing useful stuff. Respect bro. You should be really talented.

Making simple HTML websites isn't really coding. I didn't start coding for real until I was 20. The stuff I made back then really wasn't that impressive. I was just experimenting.

**Comment by es128** [comment]
> Making simple HTML websites isn't really coding

At the risk of being pedantic here - of course it isn't programming, but I'd argue that it *is* coding. I find it to be an important semantic distinction when discussing these topics with completely non-technical people or people with interest in getting started, but hardly any knowledge or experience. I don't like the idea of telling people who have only worked with HTML and CSS so far and that what they've done is not coding. It's often the first step.

**Comment by sindresorhus** [comment]
Sure. The point was that I didn't really do any real programming before I was 20. Mostly just copy pasting and experimenting with simple HTML/CSS.

**Comment by Qix-** [comment]
http://www.govindtiwari.blogspot.com/ is futuristic compared to some of the stuff we started out on (if my math is right, Sindre and I started around the same since we're roughly 5 years apart). Imagine there not really being Javascript in the browser (it was called DHTML) and CSS was new and mostly a toy. HTML3 was the standard when I started, PHP wasn't really popular (it had just been conceived), and server sided scripting was still mostly in a folder called `cgi-bin` and written in Perl.

Here is [the website](https://www.quikwok.com/) for my favorite Chinese restaurant in my hometown. It was made in Frontpage 2.0, made probably in 1996 or '97. This was considered the norm back then.

If you look hard enough, there is a .MID embedded into the page (which modern browsers no-longer support):

```html
<bgsound src="music/shamisen.mid" loop="1">
```

I welcome you to download it, open it in Garageband or something, and tell me if you wanted to hear that on `loop="1"` while you were deciding what you wanted to eat. :)

Oh and we didn't have Google back then either. So HTML3 books from the library were my personal preference.
